pom.xml

```diff
@@ -60,31 +52,32 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
 
-        <spring-cloud.version>Greenwich.RC2</spring-cloud.version>
-
-        <spring.boot.version>2.1.1.RELEASE</spring.boot.version>
+        <spring-boot.version>2.1.1.RELEASE</spring-boot.version>
+        <spring-cloud.version>Finchley.RELEASE</spring-cloud.version>
+        <spring-cloud-starter-feign.version>1.4.6.RELEASE</spring-cloud-starter-feign.version>
```

Comment:
I don't think these are required - Spring Cloud BOM already pulls the right versions